js/packages/teams-ai/src/models/OpenAIModel.ts

```diff
@@ -120,13 +164,13 @@ export class OpenAIModel implements PromptCompletionModel {
     /**
      * Options the client was configured with.
      */
-    public readonly options: OpenAIModelOptions | AzureOpenAIModelOptions;
+    public readonly options: OpenAIModelOptions | AzureOpenAIModelOptions | OpenAILikeModelOptions;
 
     /**
      * Creates a new `OpenAIModel` instance.
      * @param {OpenAIModelOptions} options - Options for configuring the model client.
      */
-    public constructor(options: OpenAIModelOptions | AzureOpenAIModelOptions) {
+    public constructor(options: OpenAIModelOptions | AzureOpenAIModelOptions | OpenAILikeModelOptions) {
```

Comment:
Does `OpenAILikeModelOptions` also apply to Azure AI Studio? What is the extent of its applicability?

Reply:
We don't need a separate one... In AlphaWave We called it out as `OpenSourceModelOptions` but OpenAILike is actually a better descriptor as it has little to do with OSS based models and is more about conforming to OpenAI's REST contract.